### PR TITLE
Add Image as link in Pod Overview

### PR DESCRIPTION
**Pull request by dabasnidhi**
Pod for testing - http://nidabas-desk:8096/summary?namespace=default
image

### CODE REVIEW

File: src/WebUI/ImageDetails/ImageDetails.tsx [hunk 1]
```diff
@@ -125,6 +117,27 @@ export class ImageDetails extends BaseComponent<IImageDetailsProperties, IImageD
         );
     }
 
+    private _getImageDetailsRowsData(imageDetails: IImageDetails): ArrayItemProvider<any> {
+        let imageDetailsRows: any[] = [];
+        const digest: string = imageDetails.hash || "";
+        const imageType: string = imageDetails.imageType || "";
+        const mediaType: string = imageDetails.mediaType || "";
+        const registryName: string = this._getRegistryName();
+        const imageSize: string = ""; // ToDo: Size is currently not available in imageDetails.
```
Comment (madhuv-msft) [inline src/WebUI/ImageDetails/ImageDetails.tsx]: show as '-' for the time being

File: src/WebUI/ImageDetails/ImageDetails.tsx [hunk 2]
```diff
@@ -125,6 +117,27 @@ export class ImageDetails extends BaseComponent<IImageDetailsProperties, IImageD
...
+        const labels: string[] = imageDetails.tags || [];
+        const jobName: string = format("#{0} on {1}", imageDetails.buildId || "", imageDetails.buildDefinitionName || ""); // ToDo: JobName is currently not available in imageDetails.
```
Comment (madhuv-msft) [inline src/WebUI/ImageDetails/ImageDetails.tsx]: localization, also i thought this jobDetails suppose to be part of the header as per ppt.
image

File: src/WebUI/Pods/PodsDetails.tsx [hunk 3]
```diff
@@ -71,5 +83,20 @@ export class PodsDetails extends BaseComponent<IPodsDetailsProperties, IPodsDeta
         });
     }
 
+    private _showImageDetails = (imageId: string) => {
+        const imageService = KubeSummary.getImageService();
```
Comment (madhuv-msft) [inline src/WebUI/Pods/PodsDetails.tsx]: a thought, should we have a store and then use that to get the data?

**Comment by madhuv-msft** [comment]
please see if we have to handle the link in the orphan pod also, I could see in orphan pod link is not provided, and also when I click on the link of the orphan pod details, link is not working

File: src/WebUI/ImageDetails/ImageDetails.tsx (hunk 2 above)
Comment (dabasnidhi) [reply src/WebUI/ImageDetails/ImageDetails.tsx]: Shashank asked me to add the job text after labels. I will sync up with him on correct behavior

File: src/WebUI/Pods/PodsDetails.tsx (hunk 3 above)
Comment (dabasnidhi) [reply src/WebUI/Pods/PodsDetails.tsx]: Added a ToDO for the same

**Pull request merged**